src/wallet/wallet.cpp

```diff
@@ -2772,10 +2772,6 @@ bool CWallet::CreateSidechainDeposit(CTransactionRef& tx, std::string& strFail,
     // Select coins to cover sidechain deposit
     std::vector<COutput> vCoins;
     AvailableCoins(vCoins);
-//(const std::vector<COutput>& vAvailableCoins, const CAmount& nTargetValue, std::set<CInputCoin>& setCoinsRet, CAmount& nValueRet, const CCoinControl *coinControl = NULL) const;
```

Comment:
This comment cleanup doesn't belong in this PR, I will open a separate PR for this.

Reply:
Done: https://github.com/drivechain-project/bitcoin/commit/dafde109765b56f663dbf239e644a0085d021b0a